packages/rest/src/types.ts

```diff
@@ -107,6 +107,10 @@ export interface RequestBodyValidationOptions extends ajv.Options {
    * - `string[]`: Add an array of keywords from `ajv-keywords`
    */
   ajvKeywords?: true | string[];
+  /**
+   * Enable error message customizing.
+   */
+  ajvTranslate?: (errors: ajv.ErrorObject[]) => ajv.ErrorObject[];
```

Comment:
I like the idea of allowing app developers to transform the errors reported by AJV. I am little bit reluctant to limit this new config option to translations only, can find a more generic name please? For example: `ajvErrorTransformer`.

Reply:
I was uncertain about using verb or noun, I think `ajvErrorTransformer ` is good. 👍